spec/models/mixins/event_mixin_spec.rb

```diff
@@ -41,19 +41,108 @@ def event_where_clause(assoc)
   end
 
   context "event_stream_filters" do
+    # Note
+    # 1) BottleneckEvent is not including event mixin, event_where_clause references missing resource_type, resource_id on event streams
+    # 2) EmsCluster#event_where_clause does an OR:  ems_cluster_id OR (host_id in ? OR dest_host_id IN ?) OR (vm_or_template in ? OR dest_vm_or_template_id in ?) (for now we just do ems_cluster_id in ems_event_filter)
+    # 3) Host#event_where_clause does an OR: host_id OR dest_host_id... right now we only do host_id in ems_event_filter
```

Comment:
This one is actually important, so we need to figure this out (i thought OR was possible via API 🤔, but it might be only for a single attribute).  Additionally, this pretty much _only_ applies to clone and DRS events, and perhaps a few others, so it's not a very frequent case, but DRS events can be important for issue root cause analysis.

Reply:
Yeah, the problem is we do ANDs of type `EmsEvent AND thing1 AND thing2 AND host_id` and would need to do `(EmsEvent AND thing1 AND thing2) AND (host_id OR dest_host_id)`.  Maybe that's possible in the API but I couldn't get it to work before.